core/src/main/java/io/camunda/migrator/RuntimeMigrator.java

```diff
@@ -86,6 +95,14 @@ public void migrate() {
     activateMigratorJobs();
   }
 
+  protected List<String> listSkippedProcessInstances() {
+   return new Pagination<String>()
+        .batchSize(batchSize)
+        .maxCount(idKeyMapper::findSkippedProcessInstanceIdsCount)
+        .page(offset -> idKeyMapper.findSkippedProcessInstanceIds(0, batchSize))
```

Comment:
/q When skipped instances > 500 doesn't this always return the same 500 skipped instances?